web/docs/guides/auth.mdx

```diff
@@ -12,7 +12,7 @@ Supabase makes it simple to manage your users.
 <source src="/videos/auth-zoom2.mp4" type="video/mp4" muted playsInline />
 </video>
 
-When a user signs up, Supabase assigns them a unique ID. You can reference this ID anywhere in your database. For example, you might create a `profiles` table referencing the user using a `user_id` field.
+When users sign up, Supabase assign them a unique ID. You can reference this ID anywhere in your database. For example, you might create a `profiles` table referencing the user using a `user_id` field.
```

Comment:
> Supabase assign them

assigns?

Reply:
If the orga­ni­za­tion is referred to as “they” or “who,” use a plural verb with the company name. If the orga­ni­za­tion is referred to as “it” or “which,” use a singular verb. I think most of the organization takes plural.